drivers/TinyGSM/TinyGsmClient.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * file       TinyGsmClient.h
+ * author     Volodymyr Shymanskyy
+ * license    LGPL-3.0
+ * copyright  Copyright (c) 2016 Volodymyr Shymanskyy
+ * date       Nov 2016
+ */
+
+#ifndef TinyGsmClient_h
+#define TinyGsmClient_h
+
+#if defined(TINY_GSM_MODEM_SIM800) || defined(TINY_GSM_MODEM_SIM868) || defined(TINY_GSM_MODEM_U201) || defined(TINY_GSM_MODEM_ESP8266)
+#define TINY_GSM_MODEM_HAS_SSL
+#endif
+
+#if defined(TINY_GSM_MODEM_SIM808) || defined(TINY_GSM_MODEM_SIM868) || defined(TINY_GSM_MODEM_A7)
+#define TINY_GSM_MODEM_HAS_GPS
+#endif
+
+#if   defined(TINY_GSM_MODEM_SIM800) || defined(TINY_GSM_MODEM_SIM900)
+#define TINY_GSM_MODEM_HAS_GPRS
+#include "TinyGsmClientSIM800.h"
+typedef TinyGsmSim800 TinyGsm;
+typedef TinyGsmSim800::GsmClient TinyGsmClient;
+typedef TinyGsmSim800::GsmClientSecure TinyGsmClientSecure;
+
+#elif defined(TINY_GSM_MODEM_SIM808) || defined(TINY_GSM_MODEM_SIM868)
+#define TINY_GSM_MODEM_HAS_GPRS
+#include "TinyGsmClientSIM808.h"
+typedef TinyGsmSim808 TinyGsm;
+typedef TinyGsmSim808::GsmClient TinyGsmClient;
+typedef TinyGsmSim808::GsmClientSecure TinyGsmClientSecure;
+
+#elif defined(TINY_GSM_MODEM_A6) || defined(TINY_GSM_MODEM_A7)
+#define TINY_GSM_MODEM_HAS_GPRS
+#include "TinyGsmClientA6.h"
+typedef TinyGsm::GsmClient TinyGsmClient;
+
+#elif defined(TINY_GSM_MODEM_M590)
+#define TINY_GSM_MODEM_HAS_GPRS
+#include "TinyGsmClientM590.h"
+typedef TinyGsm::GsmClient TinyGsmClient;
+
+#elif defined(TINY_GSM_MODEM_U201)
+#define TINY_GSM_MODEM_HAS_GPRS
+#include "TinyGsmClientU201.h
```

Comment:
I think this line miss a " :)

Reply:
You are right! I bet it was that Jenkins character who swiped it. He seems dodgy to me, I don't trust him....